samples/subsys/video/capture_to_lvgl/prj.conf

```diff
@@ -0,0 +1,20 @@
+CONFIG_VIDEO=y
+CONFIG_VIDEO_SW_GENERATOR=y
+
+CONFIG_PRINTK=y
+CONFIG_LOG=y
+
+CONFIG_MAIN_STACK_SIZE=4096
+
+CONFIG_DISPLAY=y
+CONFIG_DISPLAY_LOG_LEVEL_ERR=y
+
+CONFIG_LVGL=y
+CONFIG_LV_MEM_CUSTOM=y
+CONFIG_LV_USE_LOG=y
+CONFIG_LV_USE_LABEL=y
+CONFIG_LV_USE_BTN=y
+CONFIG_LV_USE_ARC=y
+CONFIG_LV_USE_IMG=y
```

Comment:
given you are not using `LV_CONF_MINIMAL`, I don't think these are necessary as they already default to `y`.
That being said, it would be nice to use `LV_CONF_MINIMAL` and make sure that only the LVGL feature you need are turned on as this will significantly reduce the ROM usage.

Reply:
Is this configuration below ok? I have to keep the `CONFIG_LV_USE_IMG=y`, otherwise I get a compile error regarding the `lv_img_create` function.

```
CONFIG_LVGL=y
CONFIG_LV_CONF_MINIMAL=y
CONFIG_LV_MEM_CUSTOM=y
CONFIG_LV_USE_IMG=y
CONFIG_LV_Z_MEM_POOL_SIZE=16384
```